pokete_classes/buy.py

```diff
@@ -6,6 +6,18 @@
 from .loops import std_loop
 from .ui_elements import Box, ChooseBox
 from .inv_items import invitems
+from . import movemap as mvp
+
+
+class InvBox(Box):
+    """Box wrapper for inv"""
+
+    def resize_view(self):
+        """Manages recursive view resizing"""
+        self.remove()
+        self.overview.resize_view()
```

Comment:
Where is `.overview` defined for `self`? There's no `__init__` function here, and it feels weird to expect an externally defined attribute for a class.